components/Layout/SectionWrapper.jsx

```diff
@@ -15,10 +19,21 @@ const SectionWrapper = ({ children, customClasses = "px-8 py-12 lg:p-24", backgr
   }, [backgroundType]);
 
   return (
-    <section className={`${customClasses} ${returnBackgroundType} scroll-mt-[100px]`} id={id}>
+    <section className={cn(customClasses, returnBackgroundType, 'scroll-mt-[100px]')} id={id}>
       {children}
     </section>
   );
 };
 
+SectionWrapper.propTypes = {
+  backgroundType: PropTypes.string,
+  children: PropTypes.element.isRequired,
+  customClasses: PropTypes.string,
+  id: PropTypes.string,
+};
+SectionWrapper.defaultProps = {
+  backgroundType: undefined,
+  customClasses: 'px-8 py-12 lg:p-24',
+  id: '',
+};
```

Comment:
kept it here and removed from the Component's arguments, because prop types needs it defined this way, otherwise I have `propType "customClasses" is not required, but has no corresponding defaultProps declaration.`